Dockerfile

```diff
@@ -6,30 +17,23 @@ ADD package.json package.json
 RUN npm install
 RUN npm install -g pm2
 
-RUN mkdir /sparrow
-RUN cp -a /tmp/node_modules /sparrow
+RUN mkdir /quran
+RUN cp -a /tmp/node_modules /quran
 
-WORKDIR /sparrow
-ADD . /sparrow/
-ENV NODE_ENV production
-ENV API_URL http://marketplace.peek.com
-ENV PIRATE_URL http://www.peek.com
+WORKDIR /quran
+ADD . /quran/
 RUN npm run build
 
-# upload js and css
-WORKDIR /sparrow/build
-# UPLOAD TO S3!
+# ssh keys
+#WORKDIR /root
+#RUN mv /quran/.ssh /root/
 
-# go back to /sparrow
-WORKDIR /sparrow
+# upload js and css
+#WORKDIR /quran/build
+#RUN rsync --update --progress -raz main* ahmedre@rsync.keycdn.com:zones/assets/
 
-ENV NODE_ENV production
-ENV NODE_PATH "./src"
-ENV HOST 127.0.0.1
-ENV PORT 8000
-ENV API_URL http://marketplace.peek.com
-ENV PIRATE_URL http://www.peek.com
```

Comment:
yeah, @mmahalwy was experimenting with stuff and accidentally replaced the Dockerfile, that's why i removed it.